tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+
+```bash
+$ meilisearch
+
+# --- Expected output for v0.10.1 ---
+888b     d888          d8b 888 d8b  .d8888b.                                    888
+8888b   d8888          Y8P 888 Y8P d88P  Y88b                                   888
+88888b.d88888              888     Y88b.                                        888
+888Y88888P888  .d88b.  888 888 888  "Y888b.    .d88b.   8888b.  888d888 .d8888b 88888b.
+888 Y888P 888 d8P  Y8b 888 888 888     "Y88b. d8P  Y8b     "88b 888P"  d88P"    888 "88b
+888  Y8P  888 88888888 888 888 888       "888 88888888 .d888888 888    888      888  888
+888   "   888 Y8b.     888 888 888 Y88b  d88P Y8b.     888  888 888    Y88b.    888  888
+888       888  "Y8888  888 888 888  "Y8888P"   "Y8888  "Y888888 888     "Y8888P 888  888
+
+[2020-05-04T11:47:13Z INFO  meilisearch] Database path: "./data.ms"
+[2020-05-04T11:47:13Z INFO  meilisearch] Start server on: "127.0.0.1:7700"
+```
+
+## Step 2: Run MeiliSearch as a service
+
+In Linux environments, a `service` is a process that can be launched when the operating system is booting, and it will keep running in the background. One of its biggest advantages is that it can make your program available at any moment, in a stable and consistent way. Even if it finds some execution problem or crashes, the service will be restarted and your program will be run again.  
+
+> This is not required, but if you are new to services and systemd, you can learn the [basics of Linux services](https://www.hostinger.com/tutorials/manage-and-list-services-in-linux/) here.  
+
+On Debian (and Linux distributions in general), `systemd` allows you to create and manage your own custom services. We want to make sure that MeiliSearch is always responding to your requests. And for that, we are going to build our own service. This way, we will make sure that your MeiliSearch service will always be available and running when the server is on. If any crash occurs during the running of MeiliSearch, or the system reboots for any reason, systemd automatically restarts MeiliSearch for you.  
+
+### 2.1 Create a service file
+
+Service files are text files that tell your operating system how to run your program, and when. They live in the `/etc/systemd/system` directory, and your system will load them when it boots. In this case we will use a very simple service file that will run MeiliSearch at the port `7700`.  
+
+We are also using the `--env` flag to run MeiliSearch in a production environment, and `--master-key` to have a key that will let MeiliSearch create reading and writing keys. With this generated keys, you can easily control who can access or create new documents, indexes, or change the configuration. You can change this `Master Key` to any value you want in the following command, but you should choose a safe and random key for security purposes, never share it and, just, **keep it safe**.
```

Comment:
"You/your" is mostly used here. The perspective narrative of this page should be aligned.
```suggestion
To run MeiliSearch in a production environment, use the `--env` flag. To generate a master key that will let MeiliSearch create reading and writing keys, use the `--master-key` flag. Using them, you can easily control who can access or create new documents, indexes, or change the configuration. You can change this `Master Key` to any value you want in the following command, but you should choose a safe and random key for security purposes, never share it and, just, **keep it safe**.
```